filter/running.go

```diff
@@ -22,6 +22,21 @@ func (f running) Execute(items []types.CloudItem) []types.CloudItem {
 				log.Debugf("[RUNNING] Filter instance, because it's not in RUNNING state: %s", item.GetName())
 				return false
 			}
+		case types.Stack:
```

Comment:
What does running means for Stack and Disk?

Reply:
I copied it from the longrunning filter, but actually I was only going to use it for databases. Should I remove stack and disk?

Reply:
We can keep t, probably we're not going to invoke the filters with this combination anyways.